File: R/pcatsAPIclientR/pcats_api_example_manuscript_6B.R
library(pcatsAPIclientR)

#example 6

jobid <- pcatsAPIclientR::staticGP(datafile="../../data/example6.csv",
                   outcome="Y",
                   treatment="A",
                   x.explanatory="X",
                   x.confounding="X",
                   burn.num=500, mcmc.num=500,
                   outcome.type="Continuous",
                   method="BART"                   
                   )

cat(paste0("JobID: ",jobid,"\n"))

status <- pcatsAPIclientR::wait_for_result(jobid)

if (status=="Done") {
   cat(pcatsAPIclientR::print(jobid))
}

File: python/pcats_api_example_manuscript_6B.py
import pcats_api_client as pcats_api

#example 6

jobid=pcats_api.staticgp(datafile="../data/example6.csv", 
                         outcome='Y',
                         treatment='A',
                         x_explanatory='X',
                         x_confounding='X',
                         burn_num=500,
                         mcmc_num=500,
                         outcome_type='Continuous',
                         method='BART')

print("JobID: {}".format(jobid))

status = pcats_api.wait_for_result(jobid)

if status=="Done":
    print(pcats_api.printgp(jobid))
else:
    print("Error")

